cirq/google/engine/engine_sampler.py

```diff
@@ -64,3 +64,26 @@ def run_sweep(
     @property
     def engine(self) -> 'cirq.google.Engine':
         return self._engine
+
+
+def get_engine_sampler(processor_id: str, gate_set_name: str) \
```

Comment:
It should be possible to override the environment variable by specifying an argument here.

Reply:
addressed in #2767 ; I'll rebase once that's merged